packages/amplify-cli-core/API.md

```diff
@@ -1133,6 +1140,13 @@ export interface INotificationsResourceMeta {
 // @public (undocumented)
 export type InputOptions = Record<string, string | boolean>;
 
+// @public (undocumented)
+export type InputParameters = {
+    forcePush?: boolean;
+    iterativeRollback?: boolean;
+    restoreBackend?: boolean;
+};
```

Comment:
I think I saw something similar on @sdstolworthy PR - the one we were discussing `Record<string, string` and "core parameters" vs category specific parameters. Could you check with @sdstolworthy if this isn't duplicating his work ?

Reply:
@sobolk For these three properties, I think it's appropriate to have them separate here. Looking through the code, they do all certainly relate to similar input properties, they are explicitly set on `exeInfo` in `amplify-cli`, and referenced in one or two other packages. I think it's fine in this case to definitely type them here. But I agree, @kuhlmanp, I would look for a name other than `InputParameters` for this grouping.  `ExecutionInputOptions`, `ExecutionInputFlags` maybe?

Reply:
If typescript has anonymous types maybe that's an option, so we don't claim good names prematurely.
Otherwise I think prefixing is good option. I.e. `ExecInfoInputParameters`.
Or maybe we create namespace for these types.

Reply:
Took the namespacing option. These are good points. Thanks!